esql/challenges/querys-searchable-snapshot.json

```diff
@@ -0,0 +1,113 @@
+{
+  "name": "query-searchable-snapshot",
+  "description": "Tests for searchable snapshots",
+  "default": false,
+  "parameters": {
+    "generate-data": false
+  },
+  "schedule": [
+      {
+        "name": "register-snapshot-repository",
+        "operation": {
+            "operation-type": "create-snapshot-repository",
+            "repository": "{{ p_snapshot_repo_name }}",
+            "body": {
+                "type": {{ p_snapshot_repo_type | tojson }},
+                "settings": {{ p_snapshot_repo_settings | tojson(indent=2)}}
+            }
+       }
+    },
+    {
+      "name": "mount-searchable-snapshot",
+      "operation": {
+        "operation-type": "raw-request",
+        "method": "POST",
+        "path": "/_snapshot/logsdb-scaling-repository/logging-test/_restore"
+      }
+    },
```

Comment:
Would you mind using built-in `restore-snapshot` operation, see https://esrally.readthedocs.io/en/stable/track.html#restore-snapshot, example in [here](https://github.com/elastic/rally-tracks/blob/00ad259d60f1f19c544440b8bfdefd77f5e34bed/elastic/logs/challenges/esql-ccs-snapshot.json#L104-L119)? Also, how about using parameters instead of hard-coded values?